charts/kong/templates/_helpers.tpl

```diff
@@ -21,6 +25,18 @@ We truncate at 63 chars because some Kubernetes name fields are limited to this
 {{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" -}}
 {{- end -}}
 
+{{- define "kong.certificate.issuer.root" -}}
+{{- printf "%s-%s-%s" .Release.Name .Chart.Name "selfsigned-issuer-root" -}}
+{{- end -}}
+
+{{- define "kong.certificate.issuer.ca" -}}
+{{- printf "%s-%s-%s" .Release.Name .Chart.Name "selfsigned-issuer-ca" -}}
+{{- end -}}
+
+{{- define "kong.certificate.issuer" -}}
+{{- printf "%s-%s-%s" .Release.Name .Chart.Name "selfsigned-issuer" -}}
+{{- end -}}
+
```

Comment:
The first two are no longer used outside their definition, so they can be removed entirely. Probably something similar will be re-added for further hybrid automation.

`kong.certificate.issuer` is still in use, but since we no longer need it in multiple places (for some definitions of multiple), we can inline it within the template(s) that use it--more in the Certificate template comments.

```suggestion
```

Reply:
resolved in. 97764e4